src/locales/ar.json

```diff
@@ -74,6 +74,9 @@
       ]
     }
   },
+  "gamesPage": {
```

Comment:
I have a suggestion not only related with this code but games in general, would it be more neat if we put games inside of games Page?

Reply:
I was thinking of putting all game data in an array. But I will do it after all games are submitted. I need it for games page